php-sdk-release/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM php:7.3-cli
+
+WORKDIR /home
+
+ENV PHPDOCUMENTOR_ARCHIVE_FILE=phpDocumentor.phar
+
+ENV LDTOOLS_DIR=/home/launchdarkly/tools
+
+RUN mkdir -p ${LDTOOLS_DIR}
+
+ADD downloads/${PHPDOCUMENTOR_ARCHIVE_FILE} ${LDTOOLS_DIR}/php/${PHPDOCUMENTOR_ARCHIVE_FILE}
+
+CMD /bin/bash
```

Comment:
I don't do a lot with Docker, but I notice we don't set `CMD` for our other Docker files. I want to check whether this is intentional or something left over from testing.

Reply:
It's not super important in this case but it is intentional. The base `php:7.3-cli` image sets its CMD to `php` itself, so that if you run the container you get a PHP interpreter. That's not what our image is for (we're deriving from the `-cli` image just because we're using only command-line PHP and not a webserver), so I overrode it to just run a shell.

Reply:
Thanks for the explanation!

Reply:
Added a comment in the Dockerfile to clarify.